lib/grandpa/grandpa.go

```diff
@@ -1310,9 +1307,10 @@ func (s *Service) GetSetID() uint64 {
 
 // GetRound return the current round number
 func (s *Service) GetRound() uint64 {
+	// Tim: I don't think we need to lock in these case.  Reading an int will
+	// not produce a concurrent read/write panic
```

Comment:
That depends on how `func (gm *GrandpaModule) RoundState` is called which is the only place where this function is called.